pkg/features/kube_features.go

```diff
@@ -738,7 +731,6 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	ImmutableEphemeralVolumes:                      {Default: true, PreRelease: featuregate.Beta},
 	DefaultIngressClass:                            {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.20
 	HugePageStorageMediumSize:                      {Default: true, PreRelease: featuregate.Beta},
-	ExternalPolicyForExternalIP:                    {Default: true, PreRelease: featuregate.GA}, // remove in 1.20
```

Comment:
Based on https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecation I think we need to set `LockToDefault` for 2 releases before we can remove it.

Reply:
@andrewsykim thanks, rolling it back and setting the LockToDefault